tensorflow/lite/micro/python/interpreter/src/interpreter_wrapper.cc

```diff
@@ -231,10 +231,9 @@ InterpreterWrapper::InterpreterWrapper(
 
   for (const auto& registerer : registerers_by_name) {
     if (!AddCustomOpRegistererByName(registerer.c_str(), &all_ops_resolver_)) {
-      char strbuf[128];
-      snprintf(strbuf, sizeof(strbuf),
-               "TFLM could not register custom op via %s", registerer.c_str());
-      ThrowRuntimeError(strbuf);
+      std::string strbuf;
+      strbuf = "TFLM could not register custom op via " + registerer;
+      ThrowRuntimeError(strbuf.c_str());
```

Comment:
this can be collapsed into a single line:
```cc
ThrowRuntimeError(("TFLM could not register custom op via " + registerer).c_str());
```

Reply:
done